TTS/speaker_encoder/models/resnet.py

```diff
@@ -1,10 +1,24 @@
 import numpy as np
 import torch
+import torchaudio
 from torch import nn
 
 from TTS.utils.io import load_fsspec
 
 
+class PreEmphasis(torch.nn.Module):
```

Comment:
I'm not sure but we need to have the same spectrogram as the original implementation as we are using the checkpoint provided by them. For this reason, I implemented it in this way (and because it is very efficient,  all processing will be realized in the GPU)